core/src/evaluation/functions/aggregation/mod.rs

```diff
@@ -80,6 +84,13 @@ pub enum ValueAccumulator {
         cov: f64,
     },
     Value(ElementValue),
+    CollectMultiset {
+        list: Vec<ElementValue>,
```

Comment:
What would the trade offs be of just using `ValueAccumulator::Value(ElementValue::List)`, instead of these new variants?